worker-message-prioritization-distribution-container/src/test/java/com/github/workerframework/workermessageprioritization/redistribution/DistributorIT.java

```diff
@@ -27,12 +28,19 @@
 import java.nio.charset.StandardCharsets;
 import java.time.Duration;
 import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.concurrent.TimeoutException;
 
 // This test will use whatever distributor implementation (low level) has been set as the mainClass in the maven-jar-plugin in
 // worker-message-prioritization-distribution/pom.xml.
 public final class DistributorIT extends DistributorTestBase {
 
+    private static final String RABBIT_PROP_QUEUE_TYPE = "x-queue-type";
```

Comment:
We're redefining these constants multiple times, move them to a common location.